### ISSUE TITLE
Steals focus, doesn't give it back when done

### ISSUE THREAD

**Issue by TuggyNE** [davidrusu/cactus-releases#2]
When the sampler window comes up on macOS or Windows, it steals keyboard focus even if I'm actively typing. I signed up for sudden visual distractions that can be deferred for a few seconds, not for making input unreliable or actively breaking what I'm currently doing.

Once I input a sample, at least on macOS, the app remains active, even if I'd previously minimized the main overview window to reduce the amount of visual noise. This is an unnecessary workflow disruption in all cases; once the sampler is done it should immediately get out of the way without leaving any trace except the icon in the status bar/system tray.

**Comment by davidrusu** [comment]
I've made a few changes to how samples steal focus, It should be less disruptive now. I'll Q/A it for a day and hopefully put up a release tomorrow if it feels good.

As for the app remaining active, just close the main overview window. The app will remain active in your tray. This is a limitation of macOS that I can't seem to work around. If a window of a program is closed, focus moves to any other window that program may have open. Only if no other window is open does focus return to the previous program.

But yeah just close the main window. I'll see about adding some sort of onboarding flow to make it clear it's ok to close, you can always re-open it from the tray.

**Comment by davidrusu** [comment]
Fixed in 1.7.2, you should receive the update momentarily.

I did end up finding a fix for returning focus to your last program.

I'm curious how you find the changes after you've used it for a little while.

**Comment by TuggyNE** [comment]
Seems to be working well! Pops up without focus, promptly gives focus back once I record a sample.
